.github/workflows/ci.yml

```diff
@@ -38,25 +38,47 @@ jobs:
           profile: minimal
           toolchain: stable
           override: true
-      - uses: actions-rs/cargo@v1
+      - uses: actions/cache@v2
```

Comment:
any reason we shouldn't use `cache@v3`?

Reply:
No, it's simply because I didn't know `cache@v3` is available, I will update that.

Reply:
Done!